krita_server.py

```diff
@@ -226,8 +233,12 @@ async def f_img2img(req: Img2ImgRequest):
                                          orig_width, orig_height)
 
     output_images, info, html = modules.img2img.img2img(
-        req.prompt or collect_prompt(opt),
+        req.prompt or collect_prompt(opt, 'prompts'),
+        req.negative_prompt or collect_prompt(opt, 'negative_prompt'),
+        "None",
         image,
+        mask,
+        1,
         {"image": image, "mask": mask},
```

Comment:
I think the order of arguments is a bit wrong here. Probably should be like this. It matters for inpainting mode.
```suggestion
        image,
        {"image": image, "mask": mask},
        mask,
        1,
```